gatsby-browser.js

```diff
@@ -55,4 +55,19 @@ const onClientEntry = () => {
   window.ReactDOM = require('react-dom');
 };
 
-export { wrapPageElement, onInitialClientRender, onClientEntry };
+const shouldUpdateScroll = ({ routerProps: { location } }) => {
```

Comment:
I'm a little confused as to what this is for... It looks like it resets the scroll position on any page without a `search` query parameter. Since it affects every page on the site, we might want to look at other ways of achieving what we're looking for.

Reply:
I'd love to chat about this! There was some hesitation with this approach for that exact reason.